draft-ietf-quic-transport.md

```diff
@@ -2350,15 +2401,28 @@ Sequence:
   the value selected during the handshake comes immediately before the first
   value that a server can send.
 
+Length:
+
+: An 8-bit unsigned integer containing the length of the connection ID.  Values
+  less than 4 and greater than 18 are invalid and MUST be treated as a
+  connection error of type PROTOCOL_VIOLATION.
+
 Connection ID:
 
-: A 64-bit connection ID.
+: A connection ID of the specified length.
 
 Stateless Reset Token:
 
 : A 128-bit value that will be used to for a stateless reset when the associated
   connection ID is used (see {{stateless-reset}}).
 
+An endpoint MUST NOT send this frame if it currently requires that its peer send
+packets with a zero-length Destination Connection ID.  Changing the length of a
+connection ID to or from zero-length makes it very difficult to identify when
+the new connection ID was used.  An endpoint that is sending packets with a a
```

Comment:
I don't understand this sentence. Which is the "new connection ID"? I'm inclined to remove this sentence entirely.

Reply:
This requirement seemed to need justification, that's all.